lib/index.coffee

```diff
@@ -58,33 +64,39 @@ class DockerGC
 			return _.every(results)
 
 	getDaemonFreeSpace: () ->
-		# Ensure the image is available (if it is this is essentially a no-op)
-		@dockerProgress.pull('alpine', _.noop)
-		.then =>
-			@docker.run('alpine', [ '/bin/df', '-B', '1', '/' ])
-			.then (container) ->
-				container.logs(stdout: 1)
-			.then (logs) ->
-				new Promise (resolve, reject) ->
-					logStr = ''
-					logs
-					.on('data', (data) -> logStr += data.toString('utf-8'))
-					.on('end', () -> resolve(logStr))
-					.on('error', reject)
-			.then (spaceStr) ->
-				# First split the lines, as we're only interested in the second one
-				lines = spaceStr.trim().split(/\r?\n/)
-				if lines.length isnt 2
-					throw new Error('Coult not parse df output')
+		@baseImagePromise.then (baseImage) =>
+			# Ensure the image is available (if it is this is essentially a no-op)
+			@dockerProgress.pull(baseImage, _.noop)
+			.then =>
+				@docker.run(baseImage, [ '/bin/df', '-B', '1', '/' ])
+				.then (container) ->
```

Comment:
This can be unindented (and as a bonus will reduce the diff a bunch)

Reply:
Oh yeah, nice